src/datepicker/ngb-date.ts

```diff
@@ -0,0 +1,25 @@
+export class NgbDate {
+  static from(date: {year: number, month: number, date?: number}) {
```

Comment:
Sure, but `NgbDate` is and not user-exposed, so it wasn't necessary for now. See the public API in the `NgbDatepicker`